src/content/guides/asset-management.md

```diff
@@ -503,6 +503,106 @@ import data from './data.json';
 import { foo } from './data.json';
 ```
 
+### Customize parser of JSON modules
+
+It's possible to import any `toml`, `yaml` or `json5` file as JSON module with a [custom parser](/configuration/module/#ruleparserparse) instead of specific webpack loader.
```

Comment:
```suggestion
It's possible to import any `toml`, `yaml` or `json5` files as a JSON module by using a [custom parser](/configuration/module/#ruleparserparse) instead of a specific webpack loader.
```